presto-release-tools/src/test/java/com/facebook/presto/release/tasks/TestGenerateReleaseNotesTask.java

```diff
@@ -148,9 +148,8 @@ private GenerateReleaseNotesTask initializeTask(List<Commit> commits)
                 new GenerateReleaseNotesConfig().setVersion(VERSION));
     }
 
-    private static PullRequest loadPullRequest(String title, Person author, Person mergedBy)
+    private static PullRequest loadPullRequest(int id, String title, Person author, Person mergedBy)
```

Comment:
can you help me understand why this refactoring is needed.

Reply:
TL;DR with this change it would have been possible to find that there are duplicates in release notes.

Test case originally used to create one instance of `PullRequest` object and assign it to multiple commits.
When `GenerateReleaseNotesTask` was trying to get `PullRequests` from commits and get [dicstinct](https://github.com/prestodb/presto-release-tools/blob/master/presto-release-tools/src/main/java/com/facebook/presto/release/tasks/GenerateReleaseNotesTask.java#L140) on it, it was working fine because objects were equal by ref. But, this is not what happens in reality.
If you run the changed version of the test on original code, it will fail and you can see that code is generating duplicates. Screen shot of expected vs. actual in assert failure attached.

![test-failure](https://user-images.githubusercontent.com/10567178/119733113-10dff400-be47-11eb-87ec-c31ded78824c.png)

Reply:
I needed to generate different instances of `PullRequest` which are identical and associate them with multiple commits. To achieve that, I took the `id` out of the method as otherwise it would have incremented on each call.

Reply:
Thanks for the explanation.